fcrepo/fcrepo
# Rebuild containment index to support deleted objects

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3450

# What does this Pull Request do?
Alters the ReindexService to add the start time to the containment index for all resources and add an end time for deleted resources.

# How should this be tested?

Make some resources in Fedora.
Delete some of them.
Shut down Fedora
Blow away the indexes.
Restart Fedora
Check the containment index to see that the resources are all there and have a start time. Deleted resources also have an end time.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers
